index.js

```diff
@@ -41,7 +46,7 @@ module.exports = {
 		var handler;
 		if (arguments.length === 3) {
 			handler = function(req, res) {
-				if (req.method() === method && req.url() === url) {
+				if (req.method() === method && req.url() === _getBaseUrl(url)) {
```

Comment:
Can you please work with @emirotin and decide on how this feature should work, i.e. whether the whole URL or part of the URL gets matched